src/olympia/amo/sitemap.py

```diff
@@ -75,6 +83,12 @@ def location(self, item):
     'amo': AMOSitemap(),
     'addons': AddonSitemap(),
     'collections': CollectionSitemap(),
+    # category pages (wrapped search results of addons in a category)
+    # blocked-addons?
+    # user profiles
+    # error pages?
+    # addon review pages (+ direct?)
+    # addon privacy, eula, license, versions
```

Comment:
These comments look like your own todo list :p Do we really need to commit them?

Reply:
heh, maybe?  There's a splattering of `# TODO:` all over this file.  But I'll remove.